delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
```

Comment:
**Observations**: The SQL query construction directly includes user input, which is susceptible to SQL injection.

**Recommendations**: Use prepared statements to mitigate SQL injection risks. Here’s how you can modify the delete operation:

```php
if (isset($_GET['id'])) {
    $id = $_GET['id'];
    $stmt = $connection->prepare("DELETE FROM books WHERE id = ?");
    $stmt->bind_param("i", $id);
    $stmt->execute();
    if ($stmt->affected_rows > 0) {
        echo "Book deleted successfully!";
        header("Location: list_books.php");
        exit();
    } else {
        echo "Error: " . $stmt->error;
    }
    $stmt->close();
}
```